code/modules/jobxp/JobXPRewards.dm

```diff
@@ -244,6 +244,21 @@ mob/verb/checkrewards()
 	required_levels = list("Botanist"=10)
 	path_to_spawn = /obj/item/reagent_containers/glass/wateringcan/rainbow
 
+/datum/jobXpReward/botanist/jumpsuit
+	name = "Senior Botanist Jumpsuit"
+	desc = "An old jumpsuit with an earthy smell to it."
+	required_levels = list("Botanist"=15)
+	icon_state = "?"
+	claimable = 1
+	claimPerRound = 1
+
+	activate(var/client/C)
+		boutput(C, "The jumpsuit pops into existance!")
+		var/obj/item/I = new/obj/item/clothing/under/misc/hydroponics()
```

Comment:
```suggestion
		var/obj/item/I = new /obj/item/clothing/under/misc/hydroponics()
```
very minor style thing, we usually put a space after `new`

Reply:
Had to do this one manually, git was unhappy that two suggested changes effected the same line. I don't use git much so I'm not sure if there was a "proper" way to handle this or not.

Reply:
yeah this is all github, so no other way really